BUILD.bazel

```diff
@@ -54,3 +63,22 @@ go_binary(
     ],
     visibility = ["//visibility:public"],
 )
+
+container_push(
+    name = "push-cip-latest",
+    image = ":cip-with-gcloud",
+    format = "Docker",
+    registry = "{STABLE_IMG_REGISTRY}",
+    # repository should end with the name of the image to use.
```

Comment:
is `{STABLE_IMG_NAME}` missing here?

Reply:
Doh, that looks odd. It's not missing because the image name is already embedded into STABLE_IMG_REPOSITORY. But it's better to un-embed it to make it all more explicit.

Reply:
ah, that's inconsistent with the `container_bundle` rule above?

Reply:
Ah yes, that too. I've changed this to un-embed it.